shell/components/SortableTable/selection.js

```diff
@@ -590,13 +615,17 @@ function _execute(resources, action, args, opts = {}) {
   for ( const resource of resources ) {
     let fn;
 
-    if (opts.alt && action.altAction) {
+    if (action.singleAction) {
```

Comment:
There's no example for `singleAction`. Given that extensions should all have `invoke` / bulk is this still needed?